- Codium UnWrapper added (#3)
C0D1UM/feature-templates: iOS/CodiumUnWrapper/README.md

# Description

> CodiumUnWrapper will help you to safe unwrap Optionals 

Reviewer: ```suggestion
Safely unwrap optional types.
```

> # Supported value Types

Reviewer: ```suggestion
# Supported Value Types
```

> CodiumUnWrapper supports most of the famous value types such as String, Int, Bool, Double, Float, Array, Dictionary and ... .

Reviewer: ```suggestion
This extension supports most of the Swift build-in value types such as String, Int, Bool, Double, Float, Array, Dictionary, etc .
```

It can also check if the optional value is nil or not 

> # Extension

Reviewer: I think this section should be `Usage` instead

> to use CodiumUnWrapper you just need to copy the extension file in your app and use `unWrap` or `unWrapOr(yourDefaultValue)`  for unWrap Optionals . or if you just want to check the Optional value is nil or not you can use  `isNil` or `isNotNil` to check that. 

Reviewer: To use this extension you only need to copy the example extension code to your project.

Reviewer: Then you can give example of how to use it. Example:
```swift
let value: String?
let result = value.unwrap
```
